### ISSUE TITLE
PK sim file shows error after opening

### ISSUE THREAD

**Issue by siddhivernekar** [Open-Systems-Pharmacology/PK-Sim#2894]
My PK Sim file doesnt open. Only shows error after opening it. I have installed the newest version 11.3.2
I need access to that file. Kindly help.

Application:
PK-Sim® 11.3.208

Value cannot be null.
Parameter name: key

Stack trace:
```
at System.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at OSPSuite.Utility.Collections.Cache`2.Add(TKey key, TValue value)
   at OSPSuite.Utility.Collections.Cache`2.Add(TValue value)
   at OSPSuite.Serializer.Xml.XmlEnumerationNodeMapper`3.Deserialize(TObject objectToDeserialize, XElement element, TContext context)
   at OSPSuite.Serializer.Xml.XmlSerializer`2.TypedDeserialize(TObject objectToDeserialize, XElement outputToDeserialize, TContext context)
   at OSPSuite.Core.Serialization.Xml.DataRepositoryXmlSerializer.TypedDeserialize(DataRepository dataRepository, XElement outputToDeserialize, SerializationContext serializationContext)
   at OSPSuite.Serializer.Xml.XmlSerializer`2.Deserialize(XElement outputToDeserialize, TContext context)
   at OSPSuite.Serializer.Xml.XmlSerializer`2.Deserialize[T](XElement outputToDeserialize, TContext context)
   at PKSim.Infrastructure.Serialization.Xml.XmlReader`1.ReadFrom(XElement element, SerializationContext serializationContext)
   at PKSim.Infrastructure.Serialization.Xml.XmlSerializationManager.Deserialize[TObject](Byte[] serializationBytes, SerializationContext serializationContext)
   at PKSim.Core.Services.CompressedSerializationManager.Deserialize[TObject](Byte[] serializationBytes, SerializationContext serializationContext)
   at PKSim.Infrastructure.Serialization.ORM.Mappers.ProjectMetaDataToProjectMapper.mapFrom(ObservedDataMetaData observedDataMetaData)
   at PKSim.Infrastructure.Serialization.ORM.Mappers.ProjectMetaDataToProjectMapper.<MapFrom>b__6_0(ObservedDataMetaData x)
   at OSPSuite.Utility.Extensions.EnumerableExtensions.Each[T](IEnumerable`1 list, Action`1 action)
   at PKSim.Infrastructure.Serialization.ORM.Mappers.ProjectMetaDataToProjectMapper.MapFrom(ProjectMetaData projectMetaData)
   at PKSim.Infrastructure.Serialization.ProjectPersistor.projectFrom(ProjectMetaData projectMetaData)
   at PKSim.Infrastructure.Serialization.ProjectPersistor.Load(ISession session)
   at PKSim.Infrastructure.Serialization.WorkspacePersistor.LoadSession(ICoreWorkspace workspace, String fileFullPath)
   at PKSim.Infrastructure.Workspace.<>c__DisplayClass17_0.<OpenProject>b__0()
   at PKSim.Infrastructure.Workspace.LoadProject(Action projectLoadAction)
   at PKSim.Infrastructure.Workspace.OpenProject(String fileFullPath)
   at PKSim.Presentation.Services.ProjectTask.<>c__DisplayClass32_0.<openProjectFromFile>g__openProject|0()
   at PKSim.Presentation.Services.ProjectTask.openProjectFromFile(String projectFile, Boolean shouldStartWorker)
   at PKSim.Presentation.Services.ProjectTask.Run(StartOptions startOptions)
   at PKSim.Presentation.Presenters.Main.PKSimMainViewPresenter.Run()
   at PKSim.UI.Views.Shell.<OnLoad>b__4_0()
   at OSPSuite.Utility.Extensions.ExceptionExtensions.DoWithinExceptionHandler(Object callerObject, Action actionToExecute)
```

**Comment by Yuri05** [comment]
can you upload the project?

**Comment by siddhivernekar** [comment]
> can you upload the project?

Its saying file type is not supported here. Can you send me your mail id? 
I will share it over there.

**Comment by siddhivernekar** [comment]
> you need to zip the project first, then you will be able to upload it

[Pk model.zip](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/15025733/Pk.model.zip)

**Comment by siddhivernekar** [comment]
Hello Sir, could you recover my file? @Yuri05

**Comment by Yuri05** [comment]
[Pk model.zip](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/15043178/Pk.model.zip)
removed 1 observed data set and 1 simulation

**Comment by siddhivernekar** [comment]
Thank you so much sir.
Is there any chance this error could happen again?

**Comment by Yuri05** [comment]
Which PK-Sim version did you use when you initially created this project?
And which PK-Sim version did you use when you imported the observed data?

**Comment by siddhivernekar** [comment]
When I created the project, it was Version 11.2. And then I updated it to Version 11.3 to import the observed data

**Comment by Yuri05** [comment]
Thanks for the update.
The problem occurs very rarely, however, as we did not figure out the reason yet, it still could happen in the future.

**Comment by rwmcintosh** [comment]
I think this may be related to https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/2217.

That was a fix involving serialization that was implemented in 11.3. If the project was created in 11.2 then it would have been subject to this bug where some data would be missing in the project.
